antarest/study/storage/rawstudy/model/filesystem/root/output/simulation/simulation.py

```diff
@@ -54,10 +54,10 @@ def build(self) -> TREE:
         }
 
         if not self.simulation.error:
-            for file in ["annualSystemCost", "checkIntegrity", "simulation-comments"]:
-                file_name = f"{file}.txt"
+            for file in self.config.path.rglob("*.txt"):
+                file_name = file.name
                 if (self.config.path / file_name).exists():
```

Comment:
I guess they always exist since you iterate over them now ? I'm not sure to understand why you made changes here